wgpu-hal/src/gles/device.rs

```diff
@@ -248,8 +248,10 @@ impl super::Device {
                         return Err(crate::DeviceError::Lost.into());
                     }
                     super::BindingRegister::Textures | super::BindingRegister::Images => {
-                        let loc = gl.get_uniform_location(program, name).unwrap();
-                        gl.uniform_1_i32(Some(&loc), slot as _);
+                        // The location could be none if the uniform isn't used in the shader
```

Comment:
`gl.uniform_1_i32` can accept a None. In addition, glow is already performing this check for us, so we shouldn't bother.